src/libs/state.ts

```diff
@@ -93,12 +89,7 @@ export const authStore: Atom<AuthState> = atom<AuthState>({
   cookiesAccepted: undefined,
   fenceStatus: {},
   hasGcpBillingScopeThroughB2C: false,
-  isSignedIn: undefined,
-  oidcConfig: {
-    authorityEndpoint: undefined,
-    clientId: undefined,
-  },
-  oidcUser: undefined,
+  isSignedIn: 'uninitialized',
```

Comment:
One important side effect of this change... previously `!isSignedIn` would be true for uninitialized and signed out and `isSignedIn` would be truthy only when signed in.

Now, `!isSignedIn` is false only when signed out and `isSignedIn` is truthy when uninitialized or signed in.

Thus, where we used to have to compare `isSignedIn` to false to distinguish between uninitialized and signed out, now we have to compare `isSignedIn` to true to distinguish between uninitialized and signed in.

It looks like there are a fair number of components that rely on the old behavior: where `isSignedIn` is truthy only when signed in. Those will now behave as if the user is signed in before that's been established.

Reply:
I have gone ahead and made `isSignedIn` into a properly typed `signInStatus`. Since it is not safe to use boolean operators, I nixed the boolean type all together so it can be more clearly asserted against. Unfortunately, that change pulls in some other code owners but I can sort that out.